src/web/app/src/main/java/it/geosolutions/geostore/init/GeoStoreInit.java

```diff
@@ -71,44 +71,49 @@ public void afterPropertiesSet() throws Exception {
 
         LOGGER.info("===== Starting GeoStore services =====");
         // initialize password encoding
-        initPasswordEncoding();
```

Comment:
If I remember correctly, we had to introduce this a long time ago, after changing the PW encoding for users, to have a way to automatically update pre-existing users in the database. This was almost [11 years ago](https://github.com/geosolutions-it/geostore/commit/eed34e374bf3ab377a52151ea907ad1de91d9043).

In my opinion, after all this time, we no longer need that method to be maintained and this logic could be removed completely. What do you think @offtherailz?

Reply:
Yes, it's a very old behavior that can be removed. If we don't have problems with initialization of default users, we can to to remove it.